core/indexer/elasticsearchDatabase.go

```diff
@@ -449,3 +451,17 @@ func (esd *elasticSearchDatabase) foundedObjMap(hashes []string, index string) (
 
 	return getDecodedResponseMultiGet(response), nil
 }
+
+func (esd *elasticSearchDatabase) computeBlockSearchOrder(header data.HeaderHandler) uint32 {
+	shardIdentifier := esd.txDatabaseProcessor.createShardIdentifier(header.GetShardID())
+	stringOrder := fmt.Sprintf("%d%d", shardIdentifier, header.GetNonce())
```

Comment:
The 10th nonce of metachain will have the order 110 which will be higher than the 9th nonce of the shard 2 which will have the order 49. Does it matter for the logic that you needed from this method?

Reply:
That's right but we don't care about that. We already can have a tie breaker and sort by shard id. We only need to have a rank between items in the same shard and the same indexed bulk

Reply:
ok